projects/perception/activity_recognition/demos/online_recognition/templates/index.html

```diff
@@ -0,0 +1,14 @@
+<html>
+
+<head>
+  <title>OpenDR activity recognition demo</title>
+</head>
+
+<body style="background-color:black; text-align:center">
+  <!-- <h1>OpenDR activity recognition demo</h1> -->
+  <img src="{{ url_for('video_feed') }}" alt="OpenDR activity recognition demo"
+    style="height:98vh; max-width:98%; object-fit: contain;">
+  <!-- <a href="https://github.com/LukasHedegaard/opendr-stream-video-browser">GitHub</a> -->
```

Comment:
URLs should point to official OpenDR repo.

Reply:
You are right. The link was a commented out remnant of prior code. It was deleted in af12c3e